modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -35,6 +35,30 @@ definition {
 		}
 	}
 
+	@description = "LPS-176607 - Verify that there is a 'cancel' button being showed"
```

Comment:
I know that on Test map sheet the test scenario is : `"Verify that there is a 'cancel' button being showed"` but it will be better if we change to : `"Verify that is possible cancel Organization edition"`. Sorry, I also changed on test map sheet